### PR TITLE
lifeline now heals and adds blood

### PR DESCRIPTION
**Pull request by SonixApache**
Metabolizes half as fast as DD, so half the chance and half the healing. Because coffee, or something.
If there's a better way to add blood other than adding minuscule amounts of iron I'm open to suggestions
🆑 
 - tweak: Lifeline now heals you and replenishes a bit of blood.

### CODE REVIEW

**Comment by Sooose** [comment]
I was always a little triggered that DD + Coffee didn't inherit any of DD's effects.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -5921,6 +5921,26 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
...
+		M.heal_organ_damage(1, 0)
+	if(M.getFireLoss() && prob(25))
+		M.heal_organ_damage(0, 1)
+	if(M.getToxLoss() && prob(25))
+		M.adjustToxLoss(-1)
+	if(M.dizziness != 0)
+		M.dizziness = max(0, M.dizziness - 15)
+	if(M.confused != 0)
+		M.confused = max(0, M.confused - 5)
+	M.reagents.add_reagent ("iron", 0.1)
```
Comment (MadmanMartian) [inline code/modules/reagents/Chemistry-Reagents.dm]: Use the iron define

**Review by MadmanMartian — DISMISSED** [review]

**Comment by Probe1** [comment]
👌

**Comment by Blithering** [comment]
Couldn't you just have it directly add 0.1u of O- blood to save the iron metabolism stuff?

**Comment by SonixApache** [comment]
PJB once told me that interacting with the vessel directly is highly fucking illegal, so I dunno.

**Comment by Probe1** [comment]
He could and it was suggested but to be honest it makes me feel a little queasy to directly add the blood.

**Pull request merged**